source/jormungandr/jormungandr/scenarios/tests/new_default_tests.py

```diff
@@ -343,48 +343,62 @@ def add_pt_sections(journey):
 def get_kraken_calls_test():
     for md in ["bike", "walking", "bss", "car", "ridesharing", "taxi"]:
         req = {"origin_mode": [md], "destination_mode": [md]}
-        assert get_kraken_calls(req) == {(md, md)}
+        assert get_kraken_calls(req) == {(md, md, "indifferent")}
 
-    req = {"origin_mode": ["bss", "walking"], "destination_mode": ["walking"]}
-    assert get_kraken_calls(req) == {("walking", "walking")}
+    req = {"origin_mode": ["bss", "walking"], "destination_mode": ["walking"], "direct_path": "none"}
+    assert get_kraken_calls(req) == {("walking", "walking", "none")}
```

Comment:
It's not your change, but I think we should have `{("bss", "walking", "none")}`, or maybe `{("bss", "walking", "none"), ("walking", "walking", "none")}`, no?

Reply:
According fallback_modes.py :
"we remove bss walking and  walking bss, since they are redundant for kraken (bss includes walking)"